doc/config/README.md

```diff
@@ -413,6 +413,15 @@ It may contain the key/value pairs described below in any order:
 On the upstream interface,
 for certficate update (`KUR`) requests the reprotection mode is always **keep**.
 
+On the downstream interface, according to
+[RFC 9483 section 4.1.5](https://datatracker.ietf.org/doc/html/rfc9483#section-4.1.5),
+if a request message (e.g. of type `IR`) has MAC-based protection
+and the respective type of the outgoing response message (e.g., `IP`)
+is configured to be reprotected, any given configuration of outgoing
```

Comment:
Note that this documents the behavior before the changes now being done!
This needs to be updated.